x-pack/plugin/ml/src/main/java/org/elasticsearch/xpack/ml/job/process/logging/CppLogMessageHandler.java

```diff
@@ -283,6 +283,15 @@ private void parseMessage(XContent xContent, BytesReference bytesRef) {
             if (upstreamMessage.contains("bad_alloc")) {
                 upstreamMessage += ", process ran out of memory.";
```

Comment:
The `.` at the end of this is no longer appropriate, as the line below will add a comma.